src/current/cockroachcloud/metrics.md

```diff
@@ -40,7 +40,7 @@ The time interval selector at the top of each tab allows you to filter the view
 
 CockroachDB {{ site.data.products.standard }} and CockroachDB {{ site.data.products.advanced }} users can export metrics to the following third-party cloud sinks:
 
-- AWS Cloudwatch
+- AWS CloudWatch
```

Comment:
I think this is technically `Amazon CloudWatch`.

Reply:
Thanks for the correction! I updated it to Amazon CloudWatch. I also created [DOC-10018](https://cockroachlabs.atlassian.net/browse/DOC-10018) to correct this throughout the repo.